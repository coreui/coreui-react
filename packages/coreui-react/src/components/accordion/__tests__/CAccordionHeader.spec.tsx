import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom'
import { CAccordionHeader } from '../index'

test('loads and displays CAccordionHeader component', async () => {
  const { container } = render(<CAccordionHeader>Test</CAccordionHeader>)
  expect(container).toMatchSnapshot()
})

test('CAccordionHeader customize', async () => {
  const { container } = render(<CAccordionHeader className="bazinga">Test</CAccordionHeader>)
  expect(container.firstChild).toHaveClass('bazinga')
  expect(container.firstChild).toHaveClass('accordion-header')
  expect(container).toMatchSnapshot()
})
