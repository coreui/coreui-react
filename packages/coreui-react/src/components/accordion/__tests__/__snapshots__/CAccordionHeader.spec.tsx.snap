// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CAccordionHeader customize 1`] = `
<div>
  <div
    class="accordion-header bazinga"
  >
    <button
      class="accordion-button collapsed"
      type="button"
    >
      Test
    </button>
  </div>
</div>
`;

exports[`loads and displays CAccordionHeader component 1`] = `
<div>
  <div
    class="accordion-header"
  >
    <button
      class="accordion-button collapsed"
      type="button"
    >
      Test
    </button>
  </div>
</div>
`;
